### ISSUE TITLE
Variable substitution fails with multiple vars with long names

### ISSUE THREAD

**Issue by cfriaszapater** [apickli/apickli#156]
If I have feature files with lines like this one (with 2 variables, and the name of the first one is long, and its value is short):
```
    And response body path $.Links.Content should be /bla/`aVariableWithAVeryVeryLongName`/cont/`var2`
```

Then `var2` does not get substituted by its value.

**Comment by cfriaszapater** [comment]
I'll raise a PR to fix it, I've got it locally.

**Comment by cfriaszapater** [comment]
Fixed by #157
